shared/js/media/media_frame.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
-/* global Downsample, VideoPlayer */
+/* global Downsample, VideoPlayer, MediaUtils */
```

Comment:
Please ensure that this new dependency does not break the Camera app which also uses media_frame.js